src/assets/scss/klaro.scss

```diff
@@ -0,0 +1,278 @@
+#eu-klaro {
+  .cookie-notice {
+    background: $white;
+    .cn-decline {
+      display: none;
+    }
```

Comment:
Wouldn't we be better off using the config like this: 
https://github.com/kiprotect/klaro/blob/master/dist/config.js#L75

So the decline button doesn't get added to the DOM